tests/features/solution/add_solution.feature

```diff
@@ -37,6 +37,8 @@ Feature: "Add solution" visibility options.
       | Description       | This is a test text              |
       | Documentation     | text.pdf                         |
     And I press "Save"
-    Then the "Collection solution test 2" collection has a solution named "Collection solution add solution"
+    # The name of the solution should exist in the block of the relative content in a collection.
+    When I go to the homepage of the "Collection solution test 2" collection
+    Then I should see the link "Collection solution add solution"
```

Comment:
Well I think you are right because there are no other tests that actually validate that clicking a link does not break the page of a solution so I am adding en extra two lines of testing here as a proof of this.